config/interpolate_funcs_test.go

```diff
@@ -310,6 +310,54 @@ func TestInterpolateFuncMin(t *testing.T) {
 	})
 }
 
+func TestInterpolateFuncPow(t *testing.T) {
+	testFunction(t, testFunctionConfig{
+		Cases: []testFunctionCase{
+			{
+				`${pow(1, 0)}`,
+				"1",
+				false,
+			},
+			{
+				`${pow(1, 1)}`,
+				"1",
+				false,
+			},
+
+			{
+				`${pow(2, 0)}`,
+				"1",
+				false,
+			},
+			{
+				`${pow(2, 1)}`,
+				"2",
+				false,
+			},
+			{
+				`${pow(3, 2)}`,
+				"9",
+				false,
+			},
+			{
+				`${pow(-3, 2)}`,
+				"9",
+				false,
+			},
+			{
+				`${pow(2, -2)}`,
+				"0.25",
+				false,
+			},
+			{
+				`${pow(0, 2)}`,
+				"0",
+				false,
+			},
```

Comment:
make output 
`$ make test TEST='./config' TESTARGS='-run=TestInterpolateFuncPow'
==> Checking that code complies with gofmt requirements...
==> Checking AWS provider for unchecked errors...
==> NOTE: at this time we only look for uncheck errors in the AWS package
go generate $(go list ./... | grep -v /terraform/vendor/)
2017/05/18 14:28:43 Generated command/internal_plugin_list.go
go test -i ./config || exit 1
echo ./config | \
		xargs -t -n4 go test -run=TestInterpolateFuncPow -timeout=60s -parallel=4
go test -run=TestInterpolateFuncPow -timeout=60s -parallel=4 ./config 
ok  	github.com/hashicorp/terraform/config	0.007s
`